### ISSUE TITLE
how to prepare the data ?

### ISSUE THREAD

**Issue by wusuhuang** [jimth001/formality_style_transfer_with_shared_latent_space#2]
hello! can I ack you some questions about the data? In **_sls_settings_v2_FR.py_** file,
![image](https://user-images.githubusercontent.com/57821107/155558200-37d00c17-233e-4384-a5af-5bdb11c548f7.png)
how to make the **'./new_exp_fr/embedding/embedding.bpe.big.txt'** file?  And is **'./new_exp_fr/informal.train.rule'** a txt format file? Thank you very much!

**Comment by jimth001** [comment]
Hi,


In s2s-sls-rnn, we use fasttext to pretrain the embedding on Yahoo L6 corpus. The text is tokenized by nltk and converted to subwords by bpe encoding.&nbsp;The bpe codes is also produced on Yahoo Corpus.
In s2s-sls-gpt, we directly use the bpe codes and pretrained model of gpt.
"val" means validation set.&nbsp;"inform" is the source of the task, "formal" is the target of the task. It is a txt file, line by line. Each line is the source sentence of a sample in validation set.


Best,
Yunli

---Original---
From: ***@***.***&gt;
Date: Thu, Feb 24, 2022 23:49 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [jimth001/formality_style_transfer_with_shared_latent_space] how toprepare the data ? (Issue #2)




 
hello! can I ack you some questions about the data? In sls_settings_v2_FR.py file,
 
 how to make the './new_exp_fr/embedding/embedding.bpe.big.txt' file?  And is './new_exp_fr/informal.train.rule' a txt format file? Thank you very much!
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by wusuhuang** [comment]
OK, thank you very much!


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jimth001/formality_style_transfer_with_shared_latent_space"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月28日(星期一) 凌晨0:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jimth001/formality_style_transfer_with_shared_latent_space] how to prepare the data ? (Issue #2)





 Hi, 
 
 
 In s2s-sls-rnn, we use fasttext to pretrain the embedding on Yahoo L6 corpus. The text is tokenized by nltk and converted to subwords by bpe encoding.&amp;nbsp;The bpe codes is also produced on Yahoo Corpus. 
 In s2s-sls-gpt, we directly use the bpe codes and pretrained model of gpt. 
 "val" means validation set.&amp;nbsp;"inform" is the source of the task, "formal" is the target of the task. It is a txt file, line by line. Each line is the source sentence of a sample in validation set. 
 
 
 Best, 
 Yunli 
 
 ---Original--- 
 From: ***@***.***&amp;gt; 
 Date: Thu, Feb 24, 2022 23:49 PM 
 To: ***@***.***&amp;gt;; 
 Cc: ***@***.***&amp;gt;; 
 Subject: [jimth001/formality_style_transfer_with_shared_latent_space] how toprepare the data ? (Issue #2) 
 
 
 
 
   
 hello! can I ack you some questions about the data? In sls_settings_v2_FR.py file, 
   
  how to make the './new_exp_fr/embedding/embedding.bpe.big.txt' file?  And is './new_exp_fr/informal.train.rule' a txt format file? Thank you very much! 
   
 — 
 Reply to this email directly, view it on GitHub, or unsubscribe. 
 Triage notifications on the go with GitHub Mobile for iOS or Android.  
 You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&amp;gt; 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
